library/manageiq_provider.py

```diff
@@ -75,6 +75,18 @@
       - the provider api auth token
     required: true
     default: null
+  provider_verify_ssl:
+    description:
+      - whether SSL certificates should be verified for HTTPS requests between
+      ManageIQ and the provider
```

Comment:
I opened https://github.com/dkorn/manageiq-ansible-module/issues/56
This should be addressed in a separate PR IMO